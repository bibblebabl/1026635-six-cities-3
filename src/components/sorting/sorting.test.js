import React from 'react';
import renderer from 'react-test-renderer';

import Sorting from './sorting';

it(`<Sorting /> renders correctly`, () => {
  const props = {
    sorting: `Popular`
  };

  const component = renderer.create(
      <Sorting {...props} />)
  .toJSON();

  expect(component).toMatchSnapshot();
});
