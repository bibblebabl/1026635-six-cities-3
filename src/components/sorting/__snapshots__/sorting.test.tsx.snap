// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Sorting /> renders correctly 1`] = `
<form
  action="#"
  className="places__sorting"
  method="get"
>
  <span
    className="places__sorting-caption"
  >
    Sort by 
  </span>
  <span
    className="places__sorting-type"
    onClick={[MockFunction]}
    tabIndex={0}
  >
    someSort1
    <svg
      className="places__sorting-arrow"
      height={4}
      width={7}
    >
      <use
        xlinkHref="#icon-arrow-select"
      />
    </svg>
  </span>
  <ul
    className="places__options places__options--custom  "
  >
    <li
      className="places__option places__option--active"
      onClick={[Function]}
      tabIndex={0}
    >
      someSort1
    </li>
    <li
      className="places__option "
      onClick={[Function]}
      tabIndex={0}
    >
      someSort2
    </li>
    <li
      className="places__option "
      onClick={[Function]}
      tabIndex={0}
    >
      someSort3
    </li>
    <li
      className="places__option "
      onClick={[Function]}
      tabIndex={0}
    >
      someSort4
    </li>
  </ul>
</form>
`;
